app/helpers/ods.py

```diff
@@ -0,0 +1,87 @@
+from odf.opendocument import OpenDocumentSpreadsheet
+from odf.table import Table, TableColumn, TableRow, TableCell, CoveredTableCell
+from odf.style import Style
+from odf.text import P
+from odf.namespaces import OFFICENS
+
+
+class Row(object):
+    def __init__(self, **kwargs):
+        self._row = TableRow(**kwargs)
+
+    def write_cell(self, value, **kwargs):
+        if "numbercolumnsspanned" in kwargs:
+            kwargs.setdefault("numberrowsspanned", "1")
+        if "numberrowsspanned" in kwargs:
+            kwargs.setdefault("numbercolumnsspanned", "1")
```

Comment:
aye, its both or nothing, so this just ensures if one is set the other will default to 1 if not set, could rewrite as:

```python
if "numbercolumnsspanned" in kwargs or  "numberrowsspanned" in kwargs:
    kwargs.setdefault("numberrowsspanned", "1")
    kwargs.setdefault("numbercolumnsspanned", "1")
```